app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -991,9 +1029,8 @@ class SamplesHeatmapView extends React.Component {
         {
           selectedOptions: assign(this.state.selectedOptions, newOptions),
           loading: shouldRefilterData,
-          // Reset notifiedFilteredOutTaxonIds so the user will be newly
-          // notified if their manually selected taxa do not pass the new filters.
-          notifiedFilteredOutTaxonIds: new Set(),
+          // Don't re-notify the user if their manually selected taxa do not pass the new filters.
```

Comment:
very nit: s/Don't/Do not/
I personally like to avoid single quotes on comments...

Reply:
Makes a lot of sense, I'll keep that in mind for the future!